index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;1,100&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <title>Wiinnie's Portfolio</title>
+</head>
+<body>
+    <nav>
+        <div class="logo">
+            <a href="">John Doe</a>
+        </div>
+        <div class="menu-bar">
+            <img src="IMG1/icon.png" alt="Bars">
```

Comment:
Please ensure that the hamburger button has a `nav` tag.

`<header>`
         `nav`
            `<img src="IMG1/icon.png" alt="Bars">`
        `nav`
`</header>`